RW Error running Sequential

```
2019-06-26 16:43:06,014 [testing.randomwalk.Framework] ERROR: Error during random walk
java.lang.Exception: Error running node seq.MapRedVerify
        at org.apache.accumulo.testing.randomwalk.Module.visit(Module.java:370)
        at org.apache.accumulo.testing.randomwalk.Framework.run(Framework.java:48)
        at org.apache.accumulo.testing.randomwalk.Framework.main(Framework.java:92)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 5
        at org.apache.accumulo.testing.randomwalk.sequential.MapRedVerify.visit(MapRedVerify.java:48)
        at org.apache.accumulo.testing.randomwalk.Module$1.call(Module.java:303)
        at org.apache.accumulo.testing.randomwalk.Module$1.call(Module.java:298)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at org.apache.accumulo.fate.util.LoggingRunnable.run(LoggingRunnable.java:35)
        at java.lang.Thread.run(Thread.java:748)
```

--- a/src/main/java/org/apache/accumulo/testing/randomwalk/sequential/MapRedVerify.java
+++ b/src/main/java/org/apache/accumulo/testing/randomwalk/sequential/MapRedVerify.java
@@ -38,14 +38,14 @@ public class MapRedVerify extends Test {
     String[] args = new String[3];
     args[0] = env.getClientPropsPath();
     args[1] = state.getString("seqTableName");
-    args[2] = args[2] + "_MR";
+    args[2] = args[1] + "_MR";
 
     if (ToolRunner.run(env.getHadoopConfiguration(), new MapRedVerifyTool(), args) != 0) {
       log.error("Failed to run map/red verify");
       return;
     }
 
-    Scanner outputScanner = env.getAccumuloClient().createScanner(args[5], Authorizations.EMPTY);
+    Scanner outputScanner = env.getAccumuloClient().createScanner(args[2], Authorizations.EMPTY);
     outputScanner.setRange(new Range());
 
     int count = 0;
@@ -63,8 +63,8 @@ public class MapRedVerify extends Test {
       log.error("Gaps in output");
     }
 
-    log.debug("Dropping table: " + args[5]);
+    log.debug("Dropping table: " + args[2]);
     AccumuloClient client = env.getAccumuloClient();
-    client.tableOperations().delete(args[5]);
+    client.tableOperations().delete(args[2]);
   }
 }

